src/tests/Test.ml

```diff
@@ -148,7 +148,7 @@ let main (caps : Cap.all_caps) : unit =
            it before each test. In particular, tests that check the semgrep
            output can or should change these settings. *)
         Std_msg.setup ~highlight_setting:On ();
-        Logs_.setup_logging ~highlight_setting:On ~level:(Some Logs.Debug) ()
```

Comment:
The Debug level seems appropriate for tests because debugging is what they're about. Now that only untagged log messages are logged by default, the pollution should be reduced by tagging most debug messages.

Reply:
I don't think tests are about debugging, you run tests with the expectation of all of them passing. If one fails, then you debug. Plus, printing everything is too much, and the default debug info is likely not useful. At the end you have to run the tests again.